lib/common.bash

```diff
@@ -25,6 +25,7 @@ extract_kata_env(){
 	RUNTIME_CONFIG_PATH=$(awk '/^\[Runtime\]$/ {foundit=1} /^    Path =/ { if (foundit==1) {print $3; foundit=0} } ' <<< "$toml" | sed 's/"//g')
```

Comment:
:-) I've just realised this line would be better with a `Runtime.Config` section regexp - that would help make it clear how this is picking out a different line from the one you just added for the actual runtime Path. If you have to do any edits, could you slip that one in here mayber?